### PR TITLE
Small clarifications

### PR DESCRIPTION
**Pull request by jannefiluren**
I have tried to make backticks directly visible, reordered the beginning slightly so that the intro sentence comes first, and added a sentence about that the maintainer of the registry has to approve the PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
 Registrator is a GitHub app that automates creation of registration pull requests for your julia packages to the [General](https://github.com/JuliaRegistries/General) registry.
 
+Click [here](https://github.com/apps/registratortest/installations/new) to install the app.
+
 #### How to use
 
 1) Using a Pull Request:
 
-Create a pull request on the package repo with your project file changes. Add "@JuliaRegistrator `register()`" as the content body of the pull request if you are a collaboarator on the package repository. If you are not a collaborator ask someone who is to comment "@JuliaRegistrator `register()`" on the Pull Request. This will make Registrator add a pull request to General by looking at your pull request branch.
+Create a pull request on the package repo with your project file changes. Add "@JuliaRegistrator \`register()\`" as the content body of the pull request if you are a collaboarator on the package repository. If you are not a collaborator ask someone who is to comment "@JuliaRegistrator \`register()\`" on the Pull Request. This will make Registrator add a pull request to General by looking at your pull request branch.
```
Comment (tpapp) [inline README.md]: Maybe put the first instance in a block, eg as
```
@JuliaRegistrator `register()`
```

File: README.md [hunk 2]
```diff
@@ -2,27 +2,27 @@
...
 
 2) Using an issue:
 
-Raise an issue in the package you wish to register. Add "@JuliaRegistrator `register()`" somewhere in the content of the issue if you are a collaborator to trigger Registrator. If you are not a collaborator ask someone who is to comment "@JuliaRegistrator `register()`" on the issue. This will make Registrator add a pull request to General with the appropriate changes. Registrator will look for the project file in the master branch by default. To use a custom branch comment with `register(name-of-your-branch)`.
+Raise an issue in the package you wish to register. Add "@JuliaRegistrator \`register()\`" somewhere in the content of the issue if you are a collaborator to trigger Registrator. If you are not a collaborator ask someone who is to comment "@JuliaRegistrator \`register()\`" on the issue. This will make Registrator add a pull request to General with the appropriate changes. Registrator will look for the project file in the master branch by default. To use a custom branch comment with "@JuliaRegistrator \`register(name-of-your-branch)\`".
```
Comment (tpapp) [inline README.md]: also for the `name-of-your-branch` version

File: README.md [hunk 3]
```diff
@@ -2,27 +2,27 @@
...
 
 3) Using a commit comment:
 
-On GitHub click on a commit that you wish to register. In the comment section below say "@JuliaRegistrator `register()`". Note that you must be a collaborator in order to do this.
+On GitHub click on a commit that you wish to register. In the comment section below say "@JuliaRegistrator \`register()\`". Note that you must be a collaborator in order to do this.
 
 #### Approving pull requests on the registry
 
-Comment with "@JuliaRegistrator `approved()`" on a pull request on the registry to approve it. This will make Registrator to merge the pull request on the package (if any) and create a new tag and release. The approved PR will also be merged after that.
+This step has to be performed by the maintainer of the registry, for public packages the maintainer of the [General](https://github.com/JuliaRegistries/General) registry. Comment with "@JuliaRegistrator \`approved()\`" on a pull request on the registry to approve it. This will make Registrator to merge the pull request on the package (if any) and create a new tag and release. The approved PR will also be merged after that.
```
Comment (tpapp) [inline README.md]: this in a block too

File: README.md [hunk 4]
```diff
@@ -2,27 +2,27 @@
 
 !["amelia robot logo"](graphics/logo.png)
 
-Click [here](https://github.com/apps/registratortest/installations/new) to install.
-
 Registrator is a GitHub app that automates creation of registration pull requests for your julia packages to the [General](https://github.com/JuliaRegistries/General) registry.
 
+Click [here](https://github.com/apps/registratortest/installations/new) to install the app.
```
Comment (tpapp) [inline README.md]: a button from shields.io would be nice, I made one [here](https://github.com/JuliaComputing/Registrator.jl/issues/46#issuecomment-479455721)

**Comment by jannefiluren** [comment]
@tpapp I hope I understood your suggestions correctly, and updated the pr accordingly.

**Comment by StefanKarpinski** [comment]
@nkottary, can you review?

**Comment by nkottary** [comment]
LGTM!

**Pull request merged**